tests/hooks/test_toloka.py

```diff
@@ -15,16 +15,13 @@ def setUp(self):
             Connection(
                 conn_id='toloka_default',
                 conn_type='toloka',
-                host='https://localhost/toloka/',
-                port=443,
-                extra='{"verify": "False", "project": "AIRFLOW"}',
+                password='password',
+                extra='',
```

Comment:
[According to Connection's signature](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/models/connection/index.html#airflow.models.connection.Connection), `extra` seems to be an optional argument. Maybe we should commit this argument alltogeter?